src/formatDate.js

```diff
@@ -7,8 +7,40 @@
  *
  * @returns {string}
  */
+
+const getObjectDate = (dateParts, fromFormat) =>
+  fromFormat.reduce((objectFormat, format, i) => {
+    if (format === 'YY' && dateParts[i].length === 2) {
+      objectFormat['YYYY'] =
+        dateParts[i] < '30' ? `20${dateParts[i]}` : `19${dateParts[i]}`;
+    } else {
+      objectFormat[format] = dateParts[i];
+    }
+
+    return objectFormat;
+  }, {});
+
+const convertYear = (objectDate, toFormat) => {
+  if (!objectDate['YY'] && objectDate['YYYY'] && toFormat.includes('YY')) {
+    objectDate['YY'] = objectDate['YYYY'].slice(-2);
+  }
+};
```

Comment:
The `convertYear` function is concise and serves its purpose. However, it might be more readable to rename `objectDate['YY']` to `objectDate.YY` and `objectDate['YYYY']` to `objectDate.YYYY` for property access, since these are known properties and not dynamic keys.